src/util/emitUtils.js

```diff
@@ -273,17 +277,17 @@ function initializePerspectiveNamespace(inst, io) {
       validateIp(socket),
       validateTokenOldFormat(socket),
     )
-    .then(() => {
-      pubSubStats.trackConnect();
-      socket.on('disconnect', () => {
-        pubSubStats.trackDisconnect();
-      });
-    })
-    .catch((err) => {
-      pubSubStats.trackAuthError();
-      socket.emit('auth error', err.message);
-      socket.disconnect();
-    })
+      .then(() => {
```

Comment:
formatting

Reply:
I'm just accepting the formatting changes from the ide... we don't have linter on this project... I'll add one in another PR and reformat per style guide